core/arch/arm/kernel/ldelf_loader.c

```diff
@@ -82,9 +83,12 @@ TEE_Result ldelf_load_ldelf(struct user_mode_ctx *uctx)
 	memcpy((void *)code_addr, ldelf_data, ldelf_code_size);
 	memcpy((void *)rw_addr, ldelf_data + ldelf_code_size, ldelf_data_size);
 
+	prot = TEE_MATTR_URX;
+	if (IS_ENABLED(CFG_CORE_BTI))
+		prot |= TEE_MATTR_GAURDED;
```

Comment:
`GUARDED`

Reply:
Sorry, my bad. I made the change but in the wrong commit. Will correct it.